.clang-format

```diff
@@ -1 +1,2 @@
 BasedOnStyle: Chromium
+AllowShortCaseLabelsOnASingleLine: true
```

Comment:
I think it might be best to remove this too and just exclude the few switch/case blocks where we want single line.  It doesn't seems like we want this in the general case.

Reply:
Done